Directory.Build.props

```diff
@@ -3,31 +3,11 @@
 <Project>
 
   <PropertyGroup>
-    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
-
-  <Import Project="build\NuGet.props"/>
-  <Import Project="build\Versions.props"/>
-
-  <PropertyGroup>
-    <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
-    <RepoToolsetDir>$(NuGetPackageRoot)roslyntools.repotoolset\$(RoslynToolsRepoToolsetVersion)\tools\</RepoToolsetDir>
-    <RestoreSources>
-      https://dotnetfeed.blob.core.windows.net/dotnet-core/packages/index.json;
-      https://dotnet.myget.org/F/nuget-build/api/v3/index.json
-    </RestoreSources>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <ArtifactsDir>$(DOTNET_SDK_ARTIFACTS_DIR)</ArtifactsDir>
-    <ArtifactsDir Condition="'$(ArtifactsDir)' == ''">$(RepoRoot)artifacts\</ArtifactsDir>
-    <ArtifactsDir>$([MSBuild]::EnsureTrailingSlash($(ArtifactsDir)))</ArtifactsDir>
-    <DOTNET_INSTALL_DIR Condition="'$(DOTNET_INSTALL_DIR)' == ''">$(RepoRoot)artifacts\.dotnet\$(DotNetCliVersion)\</DOTNET_INSTALL_DIR>
```

Comment:
Same for this?

Reply:
Yes, the install dir environment variable is set by build.ps1, if not set already.

---
In reply to: [184185819](https://github.com/dotnet/sdk/pull/2178#discussion_r184185819) [](ancestors = 184185819)

Reply:
Doesn't this break the scenario of building the solution from the command line, directly, rather than going through the build scripts?